frontend/src/lib/components/UserLandingPage.svelte

```diff
@@ -18,23 +18,33 @@ import UserDataInput from "./UserDataInput.svelte";
 onMount(async () => {
 	await refreshUser();
 });
+
+let windowWidth = $state(1920);
+let smallScreen = $derived(windowWidth < 800);
 </script>
+<svelte:window bind:innerWidth={windowWidth} />
 
 {#if get(currentUser)}
 	{#if get(currentUser)?.is_verified === true}
 		<div class="m-2 p-2">
-			<Tabs tabStyle="pill">
+			<Tabs tabStyle="underline">
 				<TabItem open={true}>
 					<div slot="title" class="flex items-center gap-2 text-lg">
 						<ProfileCardSolid size="lg" />
-						Persönliche Daten
+						{#if smallScreen === false}
+							{$_("userData.label")}
+						{/if}
 					</div>
 					<svelte:component this={UserDataInput} />
 				</TabItem>
-				<TabItem>
+				<TabItem onclick = {() =>{
+					activeTabChildren.set("childrenGallery");
+				}}>
 					<div slot="title" class="flex items-center gap-2 text-lg">
 						<GridPlusSolid size="lg" />
-						Kinder
+						{#if smallScreen === false}
+							{$_("childData.overviewLabel")}
+						{/if}
```

Comment:
I think you should be able to use tailwind classes for this, e.g. `class="invisible md:visible`: https://tailwindcss.com/docs/visibility#breakpoints-and-media-queries

Reply:
good point, thx.

Reply:
thx. went with `hidden md:inline`  to make the invisible elements not occupy space in the layout.